### PR TITLE
JIRA-123 - Adding UI tests for 

### PR DESCRIPTION
**Pull request by RahulVarma23**

### CODE REVIEW

File: src/practice1/Test1.java [hunk 1]
```diff
@@ -16,6 +16,8 @@ public static void main(String[] args) {
 	   char b = (char) a;
 	   
 	   System.out.println(b);
+
```
Comment (RahulVarma23) [inline src/practice1/Test1.java:19]: remove this extra space

File: src/practice1/Test3.java [hunk 2]
```diff
@@ -0,0 +1,10 @@
+package practice1;
+
+public class Test3 {
+
+	public static void main(String[] args) {
+		System.out.println("Test3");
```
Comment (RahulVarma23) [inline src/practice1/Test3.java:6]: modify this.

File: src/practice1/Test1.java (hunk 1 above)
Comment (RahulVarma23) [reply src/practice1/Test1.java:19]: this change done!

File: src/practice1/Test3.java (hunk 2 above)
Comment (RahulVarma23) [reply src/practice1/Test3.java:6]: done

**Review by Pooja0K — COMMENTED** [review]
review completed looks fine

**Pull request merged**